src/apps/ociodisplay/main.cpp

```diff
@@ -758,13 +736,52 @@ int main(int argc, char **argv)
     glutKeyboardFunc(Key);
     glutSpecialFunc(SpecialKey);
     glutDisplayFunc(Redisplay);
+
+    if(g_verbose)
+    {
+        std::cout << std::endl
+                  << "GL Vendor:    " << glGetString(GL_VENDOR) << std::endl
+                  << "GL Renderer:  " << glGetString(GL_RENDERER) << std::endl
+                  << "GL Version:   " << glGetString(GL_VERSION) << std::endl
+                  << "GLSL Version: " << glGetString(GL_SHADING_LANGUAGE_VERSION) << std::endl;
+
+        if(!g_filename.empty())
+        {
+            std::cout << std::endl;
+            std::cout << "Image: " << g_filename << std::endl;
+        }
+        std::cout << std::endl;
+        std::cout << "OIIO Version: " << OIIO_VERSION_STRING << std::endl;
+        std::cout << "OCIO Version: " << OCIO::GetVersion() << std::endl;
+    }
+
+    OCIO::ConstConfigRcPtr config;
+    try
+    {
+        config = OCIO::GetCurrentConfig();
+    }
+    catch(...)
+    {
+        const char * env = getenv("OCIO");
+        std::cerr << "Error loading the config file: '" << (env ? env : "") << "'";
+        exit(1);
+    }
+
+    if(g_verbose)
+    {
+        const char * env = getenv("OCIO");
+
+        if(env && *env)
+        {
+            std::cout << std::endl;
+            std::cout << "OCIO Configuration = '" << env << "'" << std::endl;
```

Comment:
[minor] Same comment as for ocioconvert for consistency with ":"

Reply:
Fixed